NewFile.txt

```diff
@@ -0,0 +1,7 @@
+This is a new file.
+
+With some new content.
+
+Seriously...
```

Comment:
Testing reply.

---
In reply to: [68499036](https://github.com/kivancmuslu/GitHubTest/pull/1#discussion_r68499036) [](ancestors = 68499036)